src/lwc/gsLabelMapper/gsLabelMapper.js-meta.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<LightningComponentBundle xmlns="http://soap.sforce.com/2006/04/metadata">
+    <apiVersion>50.0</apiVersion>
+    <isExposed>false</isExposed>
+    <masterLabel>NPSP Getting Started Label Mapper</masterLabel>
+    <description>Used to get value of the label in Getting Started</description>
```

Comment:
Now that I understand where these are actually used, let's change the description to better document usage:
```suggestion
    <description>Maps Custom Labels to vars that are used to generate the Getting Started page sections. These are what the Label Api Names in the GS_AdminChecklistSection and GS_AdminChecklistItem objects reference.</description>
```